src/common/lib/api.ts

```diff
@@ -143,7 +153,8 @@ export default {
   connectPeer,
   setSetting,
   swr: {
-    getAccountInfo: swrGetAccountInfo,
+    useAccountInfoCached: useAccountInfoCached,
+    mutateAccountInfo: mutateAccountInfo,
```

Comment:
I don't think these methods should live in an "swr" object (I know it was originally like this, I don't think it's the right approach). We can use swr with any of the api methods. Also, I don't think we should introduce mutation functions here that didn't exist, especially if they only mutate the cached SWR data.

Reply:
By introducing swr we split the functionality of initial fetching on account change (in AccountContext) and  updating/mutating (in any other component that triggers an update/mutation). Hence there need to be two functions.
I will remove the "swr" nesting, to me it also seemed strange.

Reply:
another option is that you can use the mutate function from the useSWR hook in the component that needs to update it. I think we should have a separation from swr and the API methods. SWR is just an optional way to retrieve data, it shouldn't be mixed in with actual API methods.